components/PostCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Post } from '../interfaces/post';
+
+interface IPostProps {
+    post: Post;
+}
```

Comment:
our interface naming standards basing on ther files is `PropsInterface`, so just rename this to `PropsInterface`